sdks/python/apache_beam/examples/complete/game/leader_board_it_test.py

```diff
@@ -53,7 +53,7 @@
 class LeaderBoardIT(unittest.TestCase):
 
   # Input event containing user, team, score, processing time, window start.
-  INPUT_EVENT = 'user1,teamA,10,%d,2015-11-02 09:09:28.224'
+  INPUT_EVENT = b'user1,teamA,10,%d,2015-11-02 09:09:28.224'
```

Comment:
I am not sure if this should be bytes. Does `self.pub_client.publish` require bytes? If so, we should encode before passing data to that method.

Reply:
yes `self.pub_client.publish` requires bytes. In this case I think specifying the input event as bytes is what is expected because it is directly passed to the client. What do you think?

Reply:
So the question is then, to the humans who read and edit this code, should INPUT_EVENT be a textual date or encoded data? I think it's easier to consider it text, up until it's time to feed it to pubsub, then when we can encode it to bytes.

I'd keep INPUT_EVENT as is change publishing to: 

```
event = self.INPUT_EVENT % self._test_timestamp
self.pub_client.publish(event.encode('utf-8'))
```
I would expect users to follow a similar pattern in their pipelines, and they might refer to beam examples for guidance, so I suggest to change this.